yaml/buildings.yaml

```diff
@@ -106,6 +106,9 @@ filters:
   # address points
   - filter:
       "addr:housenumber": true
+      way:
```

Comment:
We should have already picked up anything with a `building` tag in an earlier filter. And the `building_standard_properties` includes the house number and street, so it should be only the points that we need to filter in at this stage.